core/src/miner/miner.rs

```diff
@@ -529,11 +637,10 @@ impl Miner {
         let tx_total = transactions.len();
         let mut invalid_tx_users = HashSet::new();
 
-        let immune_users = self.immune_users.read();
```

Comment:
@sgkim126 
By removing this, the deadlock fixed.

Reply:
:+1: